spec/integrations/document_spec.rb

```diff
@@ -25,4 +25,34 @@
       expect(response['id']).not_to be_nil
     end
   end
+
+  describe '#find' do
+    let(:applicant_id) { '1030303-123123-123123' }
+    let(:document_id) { '1212121-123123-123123' }
+
+    it 'returns expected document' do
+      response = document.find(applicant_id, document_id)
+      expect(response['id']).not_to be_nil
+    end
+  end
+
+  describe '#all' do
+    let(:applicant_id) { '1030303-123123-123123' }
+
+    it 'returns expected document' do
+      response = document.all(applicant_id)
+      expect(response['documents']).not_to be_empty
```

Comment:
Could we check that an ID we're expecting is here? If not, could we change the example name as we're just checking that it returns *any* document, not the expected one(s)?